src/vaadin-crud.html

```diff
@@ -376,13 +413,29 @@ <h3 slot="header">[[__editorHeader]]</h3>
           }
         }
 
+        __onEditorPositionChange(editorPosition, currentEditorPosition) {
+          if (CrudElement._isValidEditorPosition(editorPosition)) {
+            return;
+          }
+          this.editorPosition = '';
+        }
+
+        static _isValidEditorPosition(editorPosition) {
+          return ['bottom'].indexOf(editorPosition) != -1;
```

Comment:
Should it work with the values like `b`, `bot`? Should we compare the strings over here?